catalogue/webapp/components/WorkHeader/WorkHeader.tsx

```diff
@@ -111,12 +111,14 @@ const WorkHeader: FunctionComponent<Props> = ({ work }) => {
               </Space>
             )}
 
-            {collectionManifestsCount > 0 && (
+            {Boolean(
+              collectionManifestsCount && collectionManifestsCount > 0
+            ) && (
               <Space v={{ size: 'm', properties: ['margin-top'] }}>
                 <p className={font('intb', 5)} style={{ marginBottom: 0 }}>
                   <Number
                     backgroundColor="yellow"
-                    number={collectionManifestsCount}
+                    number={collectionManifestsCount as number}
```

Comment:
Is this because `collectionManifestsCount` can be undefined? But we know it's greater than `0` at this point? Out of interest, if you rewrite the `Boolean(…)` above to an `if` statement does this casting requirement go away?

Reply:
That's the reason for it yes. Don't think I can make it an if statement